Dockerfile

```diff
@@ -117,20 +109,10 @@ RUN echo "" >> ~/.bashrc && \
 
 # Install Composer
 RUN curl -sSL https://getcomposer.org/installer | php -- --filename=composer --install-dir=/usr/bin
-RUN echo 'export PATH="$HOME/.composer/vendor/bin:$PATH"' >> ~/.bashrc
```

Comment:
Could you explain this change?

Reply:
I rather we use code analysis tools using local dependencies instead of global, now we have to maintain pipeline-apps, devtools. It would be better (in my opinion) to use local dependencies for bitbucket-pipelines and your IDE.

Reply:
OK. Let's have a discussion about these opinionated changes on Monday before we do anything with this PR. Because this would have a lot of impact.

Reply:
👍